examples/RigidBodyCases/RodRigidBodyContact/rod_cylinder_contact_friction.py

```diff
@@ -0,0 +1,279 @@
+import numpy as np
+import sys
+
+sys.path.append("../../../")
+from elastica import *
+from post_processing import plot_velocity, plot_video_with_surface
+
+
+def rod_cylinder_contact_friction_case(pulling_force=0.0, POST_PROCESSING=False):
+    class RodCylinderParallelContact(
+        BaseSystemCollection, Constraints, Connections, CallBacks, Forcing
+    ):
+        pass
+
+    rod_cylinder_parallel_contact_simulator = RodCylinderParallelContact()
+
+    # time step etc
+    final_time = 10.0
+    time_step = 1e-4
+    total_steps = int(final_time / time_step) + 1
+    rendering_fps = 30  # 20 * 1e1
+    step_skip = int(1.0 / (rendering_fps * time_step))
+
+    base_length = 0.5
+    base_radius = 0.1
+    density = 1750
+    E = 3e5
+    poisson_ratio = 0.5
+    shear_modulus = E / (2 * (1 + poisson_ratio))
+    n_elem = 50
+    nu = 0.5
+    start = np.zeros((3,))
+    direction = np.array([0, 0.0, 1.0])
+    normal = np.array([0.0, 1.0, 0.0])
+
+    rod = CosseratRod.straight_rod(
+        n_elem,
+        start,
+        direction,
+        normal,
+        base_length,
+        base_radius,
+        density,
+        nu,
+        E,
+        shear_modulus=shear_modulus,
+    )
+
+    rod_cylinder_parallel_contact_simulator.append(rod)
+    # rod.velocity_collection[0, :] -= 0.2
+
+    # Apply uniform forces on the rod
+    rod_cylinder_parallel_contact_simulator.add_forcing_to(rod).using(
+        UniformForces, force=1.0 * pulling_force, direction=direction
```

Comment:
set `pulling_force = gamma * normal_force`? then as gamma crosses coeff_friction the rod will move?